app/uk/gov/hmrc/transitmovements/models/Departure.scala

```diff
@@ -22,6 +22,7 @@ import java.time.OffsetDateTime
 
 case class Departure(
   _id: DepartureId,
+  movementType: MovementType,
```

Comment:
What happens if we do this?

```scala
case class Departure(_id: DepartureId, enrolementEORINumber: EORINumber, ...) {
    val movementType = MovementType.Departure
}
```

Will the various models include the movement type then? Would save having to add this to a departure each time.

(Alternatively, is this going to look the same as an arrival, barring the `movementReferenceNumber`? We might need to think on that, because if so, this could just be a `Movement`, in which case the variable movement type _is_ important.)

Reply:
Maybe have a Movement class that the Departure and Arrival inherit from, where Movement would contain all of the shared variables

Reply:
yes, Departure will be renamed to Movement